src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py

```diff
@@ -129,23 +129,23 @@ def acr_build_task_run(
     else:
         if queued_build:
             build_id = queued_build.build_id
-            print("Queued a build with build-id: {}.".format(build_id))
-            print("Starting to stream the logs...")
+            print("Queued a build with build-id: {}\nWaiting for a build agent...".format(build_id))
```

Comment:
Why do we want to combine the two print? Do we need to handle different line endings?

Reply:
probably just want to replace "Starting to stream the logs" with "waiting for a build agent"